src/components/landingHero/LandingHero.module.scss

```diff
@@ -1,100 +1,134 @@
 @use "../../index.module";
 
 .landingHero {
-    height: 100vh;
+    height: auto;
```

Comment:
Please do you really need to add this property? if it works well without the height, then remove it